tests/frontend/test_kibana_app/cypress/integration/features/management/settings/api-configurations.feature

```diff
@@ -1,23 +1,23 @@
-Feature: add, delete api configurations and switch between them
+Feature: API configuration instructions and connection validation
 
   As a kibana user
-  i want to change API configuration settings
-  in order to point different APIs
+  I want to check API configuration settings
+  in order to see the instructions and test the connection
 
   Scenario: Check API configuration connection
-    Given The kibana admin user is logged in using xpack authentication
+    Given The kibana admin user is logged in using odfe authentication
```

Comment:
why not use table cases for both security options
```cucumber
Given The kibana admin user is logged in using this authentication
| odfe |
| xpack |
...
```

Reply:
The goal of having different login methods is to execute different suites with those methods.

We could execute this in the same run, but everyone who runs the tests would have to configure more than one environment in order to run any test.